diem-move/diem-transactional-test-harness/src/diem_test_harness.rs

```diff
@@ -277,6 +289,31 @@ fn parse_named_private_key(s: &str) -> Result<(Identifier, Ed25519PrivateKey)> {
     Ok((name, private_key))
 }
 
+impl SignerAndKeyPair {
+    fn parse(s: &str) -> Result<Self> {
+        if let Ok(address) = RawAddress::parse(s) {
+            return Ok(Self {
+                address,
+                private_key: None,
+            });
+        };
+
+        let before_after = s.split('=').collect::<Vec<_>>();
+
+        if before_after.len() != 2 {
+            bail!("Invalid signer and key pair. Must be of the form <signer addr>=<private_key> or <named signer addr>, but found '{}'", s);
+        }
```

Comment:
Not necessary, but some fun if you want it. Could do something like 
```
let (addr_string, key_string) = match before_after[..] {
  [addr_string, key_string] => (addr_string, key_string)
  _ => bail!(...)
}
```
worth playing with, but not blocking let's just land this